src/components/ResizableBox/ResizableBox.scss

```
@@ -4,7 +4,7 @@
     align-items: center;
 
     /* stylelint-disable-next-line declaration-no-important */
-    width: 18px !important;
+    width: 3px !important;
```

You also have to add a `right` property to stick the handle to the SecondaryNavigation otherwise, there is a little space between both. It will become easier to grab it.
```suggestion
    width: 3px !important;
    right: -3px !important;
```